torch/onnx/symbolic_opset13.py

```diff
@@ -109,6 +109,136 @@ def unsafe_split_with_sizes(g, self, split_sizes, dim, _outputs=None):
     return split_with_sizes(g, self, split_sizes, dim, _outputs)
 
 
+@symbolic_helper.parse_args("v", "v", "i", "i")
+def tensor_split(g, self, indices_or_sections, dim, _outputs=None):
+    axis = g.op("Constant", value_t=torch.tensor(dim, dtype=torch.long))
+    axis = opset11.unsqueeze(g, axis, 0)
+
+    def static_tensor_split(self, indices):
+        start = g.op("Constant", value_t=torch.tensor([0], dtype=torch.long))
+        res = []
+        for i in range(_outputs - 1):
+            end = g.op("Gather", indices_or_sections, i, axis)
+            res.append(g.op("Slice", self, start, end, axis))
+            start = end
+
+        end = symbolic_helper._size_helper(g, self, axis)
+        res.append(g.op("Slice", self, start, end, axis))
+        return res
+
+    if not symbolic_helper._is_split_static(indices_or_sections, _outputs):
+        is_scalar = symbolic_helper._get_tensor_rank(indices_or_sections) == 0
+
+        if (
+            symbolic_helper._is_tensor(indices_or_sections) and not is_scalar
+        ):  # 1-d tensor
+            # loop conditions
```

Comment:
This comment is not adding anything beyond what the variable names already make clear. Please remove it.